src/components/ExploreStays/ExploreStays.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import PlacesFilter from '../PlaceFilter/PlacesFilter'
+import PlaceList from '../PlaceList/PlaceList'
+import PlaceListHeader from '../PlaceListHeader/PlaceListHeader'
+import PopularPlaces from '../PopularPlaces/PopularPlaces'
+
+function ExploreStays() {
+    return (
+        <>
+            <div className="col7 explore__cards">
+                <PlacesFilter />
+
+                    <PlaceListHeader placeName="Bengaluru" />    
```

Comment:
this could easily be an array of JSX where Bangalore and KL become items of the array - very easy to add more if that exists